apps/webpackEntryPoints.js

```diff
@@ -54,6 +54,7 @@ const CODE_STUDIO_ENTRIES = {
   'programming_expressions/show': './src/sites/studio/pages/programming_expressions/show.js',
   'sessions/lockout': './src/sites/studio/pages/sessions/lockout.js',
   'devise/sessions/new': './src/sites/studio/pages/devise/sessions/new.js',
+  'devise/sessions/_login': './src/sites/studio/pages/devise/sessions/_login.js',
```

Comment:
wow I'm surprised we didn't have JS on that page 😅

Reply:
Right!?